doc/advanced-configuration/Services.md

```diff
@@ -1,80 +1,101 @@
-    * `stanza`: Adds `send_message_chat`, `send_message_headline`, `send_stanza_c2s`
-    * `stats`: Adds `stats`, `stats_host`
-    * `vcard`: Adds `get_vcard`, `get_vcard2`, `get_vcard2_multi`, `set_vcard`, `set_vcard2`, `set_vcard2_multi`
-    * `gdpr`: Adds `retrieve_personal_data`
-    * `upload` : Adds `http_upload`
+# service_admin_extra
 
-#### Example configuration
-` {service_admin_extra, [{submods, [node, accounts, sessions]}]} `
+#### `submods`
+* **Syntax:** Array of strings representing function groups added by `service_admin_extra`.
+* **Default:** All submodules: `["node", "account", "sessions", "vcard", "gdpr",
+ "upload", "roster", "last", "private", "stanza", "stats"]`
+* **Example:** `submods = ["stats", "gdpr"]`
 
-### service_mongoose_system_metrics
+This service provides additional commands to the mongooseimctl script.
+They are bundled in the following groups:
+* `accounts`: Adds `change_password`, `check_password_hash`, `delete_old_users`,
+ `delete_old_users_vhost`, `ban_account`, `num_active_users`, `check_account`,
+  `check_password`
+* `last`: Adds `set_last`
+* `node`: Adds `load_config`, `get_cookie`, `remove_node`
+* `private`: Adds `private_get`, `private_set`
+* `roster`: Adds `add_rosteritem`, `delete_rosteritem`, `process_rosteritems`,
+ `get_roster`, `push_roster`, `push_roster_all`, `push_roster_alltoall`
+* `sessions`: Adds `num_resources`, `resource_num`, `kick_session`, `status_num_host`,
+ `status_num`, `status_list_host`, `status_list`, `connected_users_info`,
+  `connected_users_vhost`, `user_sessions_info`, `set_presence`
+* `stanza`: Adds `send_message_chat`, `send_message_headline`, `send_stanza_c2s`
+* `stats`: Adds `stats`, `stats_host`
+* `vcard`: Adds `get_vcard`, `get_vcard2`, `get_vcard2_multi`, `set_vcard`,
+ `set_vcard2`, `set_vcard2_multi`
+* `gdpr`: Adds `retrieve_personal_data`
+* `upload` : Adds `http_upload`
+
+# service_mongoose_system_metrics
 
 MongooseIM system metrics are being gathered to analyse the trends and needs of our users, improve MongooseIM, and get to know where to focus our efforts.
 See [System Metrics Privacy Policy](../operation-and-maintenance/System-Metrics-Privacy-Policy.md) for more details.
 
-#### Options
-* `report` (default: disabled) - Explicit acknowledgement that the metrics are gathered and reported.
-Enabling this option is silencing the notification reminder that metrics are gathered.
-* `no_report` (default: disabled) - When this option is set, System Metrics Service is not started and metrics are not collected.
+#### `report`
+* **Syntax:** boolean
+* **Default:** `false`
+* **Example:** `report = true`
+
+Explicit acknowledgement that the metrics are gathered and reported.
+Enabling this option silences the notification reminder that metrics are gathered.
+#### `no_report`
+* **Syntax:** boolean
+* **Default:** `false`
+* **Example:** `no_report = true`
```

Comment:
That's incorrect, we only have `report` option as having both of these is redundant. [Reference](https://github.com/esl/MongooseIM/blob/toml-config/src/config/mongoose_config_parser_toml.erl#L604)

Reply:
Well, we should have them both as they are not compliments of each other (even though the name suggests it).
 - `no_report = true` - no metrics at all, other options don't matter,
 - `no_report = false` - the default, means more or less "collect metrics"
 - `report = true` - the user acknowledges that MIM will gather reports, thus no message is shown on startup regarding metrics, 
 - `report = false` - the user will see the message on startup
It's weird, I know, but the parser shouldn't change the behaviour of the modules, right?

Reply:
I know they are not 100% complementary but part of the process was simplifying the options wherever we could have and the difference being mostly whether there would be a message or not can be more confusing than helpful. I believe that was the conclusion @chrzaszcz and I reached when discussing this matter

Reply:
Do you have the discussion somewhere? I don't care much for the issue, but it creates dead code for no `good` reason. Moreover this may be confusing as the underlying module code will surely be read more often than the parser code. It's a surprise in the future that we don't really need to inflict on ourselves.

Reply:
@chrzaszcz Can you add your opinion about that here?